Core/HLE/sceNp.cpp

```diff
@@ -18,28 +18,32 @@
 // This is pretty much a stub implementation. Doesn't actually do anything, just tries to return values
 // to keep games happy anyway.
 
-#include <mutex>
-#include <deque>
```

Comment:
nit: Since this file uses a deque, I recommend including it in the cpp file.  Even if it's also needed in an h file, the check to include once is fairly cheap.  It makes it easier later to prune down unnecessary headers in `.h` files, which can really improve overall compile time.

-[Unknown]